docs/cpu_templates/cpu-template-helper.md

```diff
@@ -0,0 +1,239 @@
+# CPU template helper tool
+
+The `cpu-template-helper` tool is a program designed to assist users with
+creating and managing their custom CPU templates.
+
+## Usage
+
+The `cpu-template-helper` tool has two sets of commands: template-related
```

Comment:
Are the fingerprint related commands only applicable or usable when using custom CPU templates?

Reply:
We don't restrict the use of the fingerprint file, but we haven't found good use cases other than custom CPU template as of now. When we find it, we can add descriptions to this doc later.

Reply:
I would still see it being usable for static templates, or even default settings if they wanted to confirm their environment doesn't change without them knowing. Not a big deal though.